typespec/package.json

```diff
@@ -7,7 +7,7 @@
     "@azure-tools/typespec-autorest": "~0.46.0",
     "@azure-tools/typespec-azure-core": "~0.46.0",
     "@azure-tools/typespec-azure-resource-manager": "~0.46.0",
-    "@azure-tools/typespec-providerhub": "^0.47.0",
```

Comment:
Should this be `~0.46.0`. I think it would be best to use all `0.46.x` versions instead of one of them randomly being `0.47.x`.

You should also run `npm ci` in this directory to update `package-lock.json`